index.html

```diff
@@ -18,7 +18,10 @@
   <div id="ui">
     <div id="header">
       <div id="title">
-        <h3>Pitt Open Data Project</h3>
+        <div id="logo">
```

Comment:
If we make the logo and the main heading a link to `index.html`, we could take out the Home link we have now in the navigation, which would be cleaner.